src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php

```diff
@@ -97,6 +97,7 @@ public function getRoutes(): array
         $listToolbarActions = [
             'sulu_admin.add',
             'sulu_admin.delete',
+            'sulu_admin.export',
```

Comment:
I think this button makes sense in most lists... Can you add it everywhere, where it would be working? What comes to mind is accounts, tags, categories, roles, snippets, ...